dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSReadWriteTest.java

```diff
@@ -45,9 +48,9 @@
 public class DFSReadWriteTest
 {
 
-    private static final String   clusterIP      = "192.168.56.101";
-    private static final String[] datasets       = { "~benchmark::integer::20kb", "~demo::example_dataset" };
-    private static final int[]    expectedCounts = { 1250, 6 };
+    private static final String   clusterIP      = "10.173.147.1";
+    private static final String[] datasets       = { "~demo::salesdata" };
+    private static final int[]    expectedCounts = { 3730};
```

Comment:
We will be loading these parameters from a properties file eventually. For now lets roll the IP and datasets back to the original for this PR.